protocol/v2/ssv/msgqueue/queue_test.go

```diff
@@ -69,15 +67,15 @@ func TestNewMsgQueue(t *testing.T) {
 		require.NoError(t, err)
 		identifier := spectypes.NewMsgID([]byte("pk"), spectypes.BNRoleAttester)
 		q.Add(generateConsensusMsg(t, spectypes.SSVConsensusMsgType, specqbft.Height(0), 1, identifier, specqbft.CommitMsgType))
-		q.Add(generateConsensusMsg(t, message.SSVDecidedMsgType, specqbft.Height(0), 1, identifier, specqbft.CommitMsgType))
+		q.Add(generateDecidedMsg(t, spectypes.SSVConsensusMsgType, specqbft.Height(0), 1, identifier, specqbft.CommitMsgType))
 		q.Add(generateConsensusMsg(t, spectypes.SSVConsensusMsgType, specqbft.Height(1), 1, identifier, specqbft.CommitMsgType))
-		q.Add(generateConsensusMsg(t, message.SSVDecidedMsgType, specqbft.Height(1), 1, identifier, specqbft.CommitMsgType))
+		q.Add(generateDecidedMsg(t, spectypes.SSVConsensusMsgType, specqbft.Height(1), 1, identifier, specqbft.CommitMsgType))
 		q.Add(generateConsensusMsg(t, spectypes.SSVConsensusMsgType, specqbft.Height(2), 1, identifier, specqbft.CommitMsgType))
-		q.Add(generateConsensusMsg(t, message.SSVDecidedMsgType, specqbft.Height(2), 1, identifier, specqbft.CommitMsgType))
+		q.Add(generateDecidedMsg(t, spectypes.SSVConsensusMsgType, specqbft.Height(2), 1, identifier, specqbft.CommitMsgType))
 
 		for i := 0; i <= 2; i++ {
 			height := specqbft.Height(i)
-			idxs := SignedMsgIndex(spectypes.SSVConsensusMsgType, identifier.String(), height, specqbft.CommitMsgType)
+			idxs := SignedMsgIndex(spectypes.SSVConsensusMsgType, identifier.String(), height, false, specqbft.CommitMsgType)
```

Comment:
if all calls passing false for decided lets remove it

Reply:
not all, e.g. `protocol/v2/ssv/validator/msgqueue_consumer.go:220`